frontend/utils/lazyLoadedView.js

```diff
@@ -1,16 +1,35 @@
 import Vue from 'vue'
-// NOTE: any modals opened with `OPEN_MODAL` go here
```

Comment:
Just a small thought: Should this comment be left in? It still seems relevant, unless it's well enough implied by this point.